modules/http/http-grammar.ym

```diff
@@ -134,6 +135,7 @@ http_tls_options
 
 http_tls_option
     : KW_CA_DIR     '(' string ')'            { http_dd_set_ca_dir(last_driver, $3); free($3); }
+    | KW_ENABLE_DEFAULT_CA_DIR '(' yesno ')'  { http_dd_set_enable_default_ca_dir(last_driver, $3); free($3); }
```

Comment:
Oops, I haven't noticed, but this `free()` should be removed.

Reply:
copy-paste :/ (I've just wanted to close this topic as fast as possible, but maybe too fast...)